### PR TITLE
Show how to generate wrappers automatically

### PR DESCRIPTION
**Pull request by yutannihilation**
Not sure about the current status of this repo, but I think this should show how to generate the wrappers automatically, even though it might be still a work in progress.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,3 +28,29 @@ library(helloextendr)
 hello()
 #> [1] "hello"
 ```
+
+## Development
+
+### Generate wrappers
+
+When you make either of the following changes, you'll need to regenerate the wrappers.
```
Comment (clauswilke) [inline README.md:36]: You may want to specify that this is about changes to the Rust source code.

**Review by clauswilke — APPROVED** [review]
Thanks! I want to add rextendr functions that help with generating wrappers, but this is good progress for now.

File: README.md (hunk 1 above)
Comment (yutannihilation) [reply README.md:36]: Good catch, thanks!

File: README.md [hunk 2]
```diff
@@ -49,10 +55,7 @@ This can be done by the following steps:
 devtools::install()
 
 # Re-generate wrappers
-brio::write_file(
-  .Call("wrap__make_helloextendr_wrappers", use_symbols = FALSE, package_name = "helloextendr"),
-  "R/extendr-wrappers.R"
-)
+rextendr::register_extendr()
 
 # Re-generate documents and NAMESPACE (If you are using RStudio, just run "Document")
```
Comment (clauswilke) [inline README.md:60]: > Re-generate *documentation*

Comment (yutannihilation) [reply README.md:60]: Oh, thanks!

**Comment by yutannihilation** [comment]
@clauswilke 
I regenerated all the files using `rextendr::use_extendr()`, except for adding `Cargo.lock` to `.gitignore`. Do you think there's anything else to improve from the default files? If there's no strong concern, I'll merge this.

**Comment by clauswilke** [comment]
Go ahead.

**Comment by yutannihilation** [comment]
Sure, thanks.

**Pull request merged**